replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+
+print(wald.test(b = coef(logit6logses), Sigma = vcov(logit6logses), Terms = 4)$result$chi2[1], digit = 3)
+```
+
+---
+
+### Study 2c
+
+> Five participants were excluded for missing responses in Study 2c, so 1,893 participants were included in the analyses.
+
+```{r echo = FALSE}
+suppressMessages(suppressWarnings(raw_2c_spss <- read.spss("framing/study_1d.sav", to.data.frame = TRUE)))
+#reads the data for study 2c
+
+clean_2c <- raw_1d_spss %>%
+  filter(filter_. != "Not Selected", go != "NA")
+
+clean_2c %>%
+  summarise(cleaned.sample.size = n())
+```
+
+> Analyses based to a median split on income revealed a more muted trend than in the previous study.  Among higher-income participants, 71%, 67%, and 51% were willing to travel when the tablet cost $300, $500, and $1,000, respectively;
+
+```{r echo = FALSE}
+clean_2c %>%
+  filter(split == 1, webercond == 0) %>%
+  summarise(threehundred.go = (sum(go) / n()) * 100)
+
+clean_2c %>%
+  filter(split == 1, webercond == 1) %>%
+  summarise(fivehundred.go = (sum(go) / n()) * 100)
+
+clean_2c %>%
+  filter(split == 1, webercond == 2) %>%
+  summarise(onethousand.go = (sum(go) / n()) * 100)
+```
+
+> among lower-income participants, the corresponding percentages were 70%, 69%, and 55%.
+
+```{r echo = FALSE}
+clean_2c %>%
+  filter(split == 0, webercond == 0) %>%
+  summarise(threehundred.go = (sum(go) / n()) * 100)
+
+clean_2c %>%
+  filter(split == 0, webercond == 1) %>%
+  summarise(fivehundred.go = (sum(go) / n()) * 100)
+
+clean_2c %>%
+  filter(split == 0, webercond == 2) %>%
+  summarise(onethousand.go = (sum(go) / n()) * 100)
+```
+
+
+```{r echo = FALSE}
+clean_2c_high_300 <- clean_2c %>%
+  filter(split == 1, webercond == 0) %>%
+  summarise(threehundred.go = (sum(go) / n()), CI = (1.96 * Sem(go))) %>%
+  round(digits = 2)
+
+clean_2c_high_500 <- clean_2c %>%
+  filter(split == 1, webercond == 1) %>%
+  summarise(fivehundred.go = (sum(go) / n()), CI = (1.96 * Sem(go))) %>%
+  round(digits = 2)
+
+clean_2c_high_1000 <- clean_2c %>%
+  filter(split == 1, webercond == 2) %>%
+  summarise(onethousand.go = (sum(go) / n()), CI = (1.96 * Sem(go))) %>%
+  round(digits = 2)
+#producing data.frames for new figure (2c) - high income
+```
+
+```{r echo = FALSE}
+clean_2c_low_300 <- clean_2c %>%
+  filter(split == 0, webercond == 0) %>%
+  summarise(threehundred.go = (sum(go) / n()), CI = (1.96 * Sem(go))) %>%
+  round(digits = 2)
+
+clean_2c_low_500 <- clean_2c %>%
+  filter(split == 0, webercond == 1) %>%
+  summarise(fivehundred.go = (sum(go) / n()), CI = (1.96 * Sem(go))) %>%
+  round(digits = 2)
+
+clean_2c_low_1000 <- clean_2c %>%
+  filter(split == 0, webercond == 2) %>%
+  summarise(onethousand.go = (sum(go) / n()), CI = (1.96 * Sem(go))) %>%
+  round(digits = 2)
+#producing data.frames for new figure (2c) - low income
+```
+
+
+```{r echo = FALSE}
+title = factor(c("Lower Income", "Higher Income"), levels = c("Lower Income", "Higher Income"))
```

Comment:
agreed - will change (it must have been getting late and I started getting sloppy...)